rpc/rpc-wrappers.go

```diff
@@ -893,13 +995,21 @@ func (cac StorageAuthorityClient) NewRegistration(reg core.Registration) (output
 	return output, nil
 }
 
-func (cac StorageAuthorityClient) NewPendingAuthorization() (id string, err error) {
-	response, err := cac.rpc.DispatchSync(MethodNewPendingAuthorization, nil)
+func (cac StorageAuthorityClient) NewPendingAuthorization(authz core.Authorization) (output core.Authorization, err error) {
+	jsonAuthz, err := json.Marshal(authz)
+	if err != nil {
```

Comment:
I'm going to leave these as-is for now, pending the rewrite of the RPC wrappers to be more consistent anyway. If you look through the rest of the file, I only log audit statements on the Server-side of the connections, not on the Client-side, where you can actually return the error up to be logged elsewhere. I chose Server simply because there, as-written, the errors would just disappear.

Anyway, it's not that I disagree, it's that I think all of the Client methods need work.